spec/services/mhv_account_type_service_spec.rb

```diff
@@ -20,6 +20,10 @@
     )
   end
 
+  before do
+    allow(Flipper).to receive(:enabled?).with(:mhv_medical_records_migrate_to_api_gateway).and_return(false)
```

Comment:
Lgtm. Moving forward if there is a test that needs `mhv_medical_records_migrate_to_api_gateway` to be `true`, I think we can include it in its context block.

Reply:
Thank you, and acknowledged. I will tidy this up when we get to that point.